template/5-mvp.md

```diff
@@ -3,18 +3,41 @@
 ## Personas and Scenarios
 
 *Who are the target personas for this product?*
+- **Event Attendees**: Students and young professionals who are interested in discovering and attending events in their area, ranging from social gatherings to professional networking events.
+- **Event Hosts/Organizers**: Students, young professionals, or organizations responsible for planning and managing events, seeking tools to streamline event registration, ticketing, and attendee engagement, as well as promot their event to a young adult audience.
 
 *Which is the key persona?*
+- **University Student**: A university student who is actively involved in campus life and interested in exploring social and academic events both on and off campus. This persona values opportunities for networking, discovering new interests, and engaging in community activities.
 
 *High-level scenarios to adopt, use and share the product.*
+- **Adopting the Product**: A university student hears about Event Radar through a campus event promotion or fellow classmates. Intrigued by the idea of easily discovering and attending events, they download the app to explore upcoming activities relevant to their academic and personal interests.
```

Comment:
suggestion:
**Adopting the Product**: A university student hears about Event Radar through a campus event promotion or fellow classmates. Intrigued by the idea of easily discovering and attending events `all through one centralized app`, they download it to explore upcoming activities relevant to their academic and personal interests.